src/elements/header/spec/script.spec.js

```diff
@@ -0,0 +1,5 @@
+import header from '../script';
+
+test('foo is exported', () => {
+  expect(header.foo).toBeDefined();
+});
```

Comment:
What do you think about a snapshot test example?
Like:
```js
test('foo returns correctly', () => {
  expect(header.foo().toString()).toMatchSnapshot();
});
```

This will create a `spec/__snapshots__` folder.